### PR TITLE
[Review] Responsive Filter Panel

### PR DESCRIPTION
**Pull request by dfosco**
Pending:

 - [x] Responsive Layout

 - [x] Add close functionality on `FilterPanel__outer` div (https://github.com/vtex-apps/filter-panel/issues/12)

---

 - [x] ~~Fix bug on Nexus 5 viewport (https://github.com/vtex-apps/filter-panel/issues/13)~~

Not able to reproduce #13 for now, will leave it to another PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "svgo": "^0.5.5",
     "svgo-loader": "^1.1.0",
     "url-loader": "^0.5.5",
+    "vtex-css-utils": "^1.2.1",
     "webpack": "^1.12.2",
```
Comment (tamorim) [inline package.json]: @dfosco we already have a vtex-css-util on this package.json. Check dependencies.

**Comment by dfosco** [comment]
@tamorim done!

File: package.json (hunk 1 above)
Comment (dfosco) [inline package.json]: @tamorim done!

**Comment by dfosco** [comment]
@tamorim @amoreira Still needs some layout adjusments, will update here for review when it's done.

**Comment by dfosco** [comment]
@tamorim Please review :zap:

**Comment by tamorim** [comment]
image

@dfosco scrollbar appearing at the bottom

**Comment by dfosco** [comment]
@tamorim Good catch! Will adjust later today. 

Remember to change the tag to [Rework] or [WIP] in cases like this ;)

**Comment by dfosco** [comment]
@tamorim Good catch! Will adjust later today. 

Remember to change the tag to [Rework] in cases like this :wink:

**Comment by dfosco** [comment]
@tamorim Done! Please _[Review]_

**Pull request merged**